src/foam/u2/wizard/views/FlexibleWizardContentsView.js

```diff
@@ -54,7 +54,7 @@ foam.CLASS({
     function render() {
       const self = this;
       const current$ = this.slot(function (data, data$currentWizardlet, data$currentSection) {
-        return data$currentSection.createView();
+        return data$currentSection?.createView() ?? this.E();
```

Comment:
that's a lotta ES6 in one place :D

Reply:
Didn't have other _options_ in this case